indico/modules/events/features/controllers.py

```diff
@@ -37,7 +38,6 @@
 class RHFeaturesBase(RHConferenceModifBase):
     def _checkParams(self, params):
```

Comment:
Looks like this method is not needed since all it does is to call the parent's class method